src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/ia/CftQueryServiceCancelTaskTest.java

```diff
@@ -71,79 +64,6 @@ void setUp() {
         );
     }
 
-    @ParameterizedTest(name = "{0}")
-    @MethodSource("getGrantTypes")
-    void should_retrieve_a_task_to_cancel(GrantType grantType) {
```

Comment:
Test data setup has commonality/duplicates for getTask(...), however the permission Types differ across the various actions (This one is Cancel). Is this really a duplicate?

Looks like we have removed this because it's IA centric. Does that mean we have the WA equivalent?